### PR TITLE
docs : better ouput with the same conformity

### PR DESCRIPTION
**Pull request by Davidffry**

### CODE REVIEW

**Comment by Davidffry** [comment]
Hello, a validation of the documentation,
Same convention is better to read the documention of installation.
but the end of the line for this, don't know.
=== "pikaur"

``` bash
pikaur -Sy trivy-bin
```

=== "yay"

``` bash
yay -Sy trivy-bin
```

## Homebrew

You can use homebrew on macOS and Linux.

```bash
brew install aquasecurity/trivy/trivy
```

**Review by knqyf263 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Our doc allows copying commands to the clipboard with one click. I think the leading `$` is bothering in such cases. But missing `$` is confusing in the case of multiple lines. So, we removed the leading `$` only when the command is one line. Any idea?

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -6,42 +6,42 @@
 === "Repository"
     Add repository setting to `/etc/yum.repos.d`.
 
-    ``` bash
-    $ sudo vim /etc/yum.repos.d/trivy.repo
+``` bash
```
[the remark this answers is not in the document]
Comment (Davidffry) [reply docs/getting-started/installation.md:9]: @knqyf263 please find my solution for : `only when the command is one line. Any idea?`

```bash
sudo tee /etc/yum.repos.d/trivy.repo << EOF
    [trivy]
    name=Trivy repository
    baseurl=https://aquasecurity.github.io/trivy-repo/rpm/releases/$releasever/$basearch/
    gpgcheck=0
    enabled=1
EOF
```

[the remark this answers is not in the document]
Comment (knqyf263) [reply docs/getting-started/installation.md:9]: >Please don't remove these indents. They are necessary for content tabs.

Could you see my comments?

**Pull request closed without merging**